src/site/content/en/progressive-web-apps/customize-install/index.md

```diff
@@ -144,38 +145,58 @@ apply different styles depending on how the app was launched. For example,
 always hide the install button and provide a back button when launched as an
 installed PWA.
 
-### Update UI based on how the PWA was launched
-
-To apply a different background color for a PWA when launched as an installed
-PWA, use conditional CSS:
-
-```css
-@media all and (display-mode: standalone) {
-  body {
-    background-color: yellow;
-  }
-}
-```
-
 ### Track how the PWA was launched
 
 To track how users launch your PWA, use `matchMedia()` to test the
-`display-mode` media query. Unfortunately, Safari on iOS doesn't support
+`display-mode` media query. Safari on iOS doesn't support
 this yet, so you must check `navigator.standalone`, it returns a boolean
 indicating whether the browser is running in standalone mode.
 
 ```js
-window.addEventListener('load', () => {
+window.addEventListener('DOMContentLoaded', () => {
+  let displayMode = 'browser tab';
   if (navigator.standalone) {
-    console.log('Launched: Installed (iOS)');
-  } else if (matchMedia('(display-mode: standalone)').matches) {
-    console.log('Launched: Installed');
-  } else {
-    console.log('Launched: Browser Tab');
+    displayMode = 'standalone-ios';
   }
+  if (window.matchMedia('(display-mode: standalone)').matches) {
+    displayMode = 'standalone';
+  }
+  // Log launch display mode to analytics
+  console.log('DISPLAY_MODE_LAUNCH:', displayMode);
 });
 ```
 
+### Track when the display mode changes
+
+To track if the user changes between standalone, and browser tab, listen for
```

Comment:
```suggestion
To track if the user changes between `standalone`, and `browser tab`, listen for
```

Formatting these as inline code because they appear to be specific JS directives